internal/acctest/config/authenticationpoliciespolicy/authentication_policies_policy_test.go

```diff
@@ -0,0 +1,290 @@
+package acctest_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+func TestAccAuthenticationPoliciesPolicy(t *testing.T) {
+	resourceName := "myAuthenticationPoliciesPolicy"
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acctest.ConfigurationPreCheck(t) },
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"pingfederate": providerserver.NewProtocol6WithError(provider.NewTestProvider()),
+		},
+		Steps: []resource.TestStep{
+			// Test simple policy
+			{
+				Config: testAccAuthenticationPoliciesPolicySimple(resourceName),
+				Check:  testAccCheckExpectedAuthenticationPoliciesPolicyAttributes(resourceName, false),
+			},
+			// Test a more complex policy
+			{
+				Config: testAccAuthenticationPoliciesPolicyComplex(resourceName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckExpectedAuthenticationPoliciesPolicyAttributes(resourceName, true),
+					resource.TestCheckResourceAttr(fmt.Sprintf("pingfederate_authentication_policies_policy.%s", resourceName), "enabled", "true"),
```

Comment:
I don't think we need checks like this for anything we explicitly set in the HCL, we need them for things that are computed that we want to verify are in place.

Reply:
added